installation/installing-images/mac.md

```diff
@@ -2,6 +2,15 @@
 
 On Mac OS you have the choice of the command line `dd` tool or using the graphical tool ImageWriter to write the image to your SD card.
 
+## Completely graphical interface
+- Connect the SD card reader with the SD card inside.
+- Install ApplePi-Baker from http://www.tweaking4all.com/hardware/raspberry-pi/macosx-apple-pi-baker/
+- Select device from list (Pi-Crust)
+- Prep the card for NOOBS (Prepare)
```

Comment:
I've not used Apple-Pi--Baker myself, but I believe if you're only writing an image file, then there's no need to "prep for NOOBS"